patterns/ecs-v1/grok-patterns

```diff
@@ -67,7 +68,7 @@ TIME (?!<[0-9])%{HOUR}:%{MINUTE}(?::%{SECOND})(?![0-9])
 DATE_US %{MONTHNUM}[/-]%{MONTHDAY}[/-]%{YEAR}
 DATE_EU %{MONTHDAY}[./-]%{MONTHNUM}[./-]%{YEAR}
 ISO8601_TIMEZONE (?:Z|[+-]%{HOUR}(?::?%{MINUTE}))
-ISO8601_SECOND (?:%{SECOND}|60)
+ISO8601_SECOND %{SECOND}
```

Comment:
This causes a regression WRT leap seconds.

Reply:
should be fine, was a duplication, [SECOND already handles `60`](https://github.com/logstash-plugins/logstash-patterns-core/blob/03adf12e98e533785347ca2d29cdaae90bb36ed6/patterns/ecs-v1/grok-patterns#L65) + there's also a spec guarding this